ModAssistant/Localisation/en.xaml

```diff
@@ -162,7 +162,7 @@
     <sys:String x:Key="Options:AllModsUninstalled">All Mods Uninstalled</sys:String>
     <sys:String x:Key="Options:CurrentThemeRemoved">Current theme has been removed, reverting to default...</sys:String>
     <sys:String x:Key="Options:ThemeFolderNotFound">Themes folder not found! Try exporting the template...</sys:String>
-    <sys:String x:Key="Options:AppDataNotFound">AppData folder not found! Try launching your game...</sys:String>
```

Comment:
The last part is good advice, maybe without the ellipsis though.

Reply:
sounds kinda passive-aggresive. it's fixed in the latest commit.